### ISSUE TITLE
Reflection Probes import support

### ISSUE THREAD

**Issue by keianhzo** [MozillaReality/hubs-blender-exporter#125]
**Description**
Reflection probes are currently imported as empties.

**To Reproduce**
- Export a glTF file with a reflection probe
- Import it again
- The imported reflection probe is an empty

**Expected behavior**
- The reflection probe element should have been imported correctly as a reflection probe Blender object with the reflection probe component.

**Comment by keianhzo** [comment]
There are a couple of issues when importing back reflection probe nodes.
1. The reflection probe is exported as just a node and imported as an empty. there is not way of just converting an object to a different type (ie Empty -> Reflection Probe) so we would need to manually create a reflection probe and completely replicate the state.
2. Even if we do the above step, the glTF doesn't export the reflection probe data so we would need to export it in the component information which I'm not sure if it's the best idea. Seems to me that the simplest way forward would be to move the relfection probe Blender object info into the component and allow adding the component to any object.

**Comment by netpro2k** [comment]
Yeah the main point of reflection probes using the Blender reflection probe objects was for their useful gizmos. Since we now have support for custom gizmos I think we want to move it to a proper component on an empty. 

We will have to do this eventually anyway since we want to add additional properties like a capture offset, box projection size, etc.

Re importing in general though, I feel like it should be fairly low priority especially for things not supported in Spoke, since that's the main _real_ usecase I can see for needing to import a gltf with component data in it. Nice to have full parity outside of just migrating out of Spoke eventually, but I suspect the amount of people who actually end up using that instead of just passing around .blend files to be pretty small.

**Comment by keianhzo** [comment]
I agree on moving the params to the component and letting this be added to any object.

On the import size, actually I saw the cloud sync being the major use case for it. Right now the Blender workflow it very tied to Spoke but eventually would be nice to have full Blender/Cloud integration and for that to be bi-directional we need glTF import support.
